lib/logsetup.js

```diff
@@ -26,19 +26,18 @@ var getTimeStamp = function() {
   return moment().format('YYYY-MM-DD HH:mm:ss Z');
 };
 
-try {
-  LOGGER.remove(LOGGER.transports.Console);
-  LOGGER.add(LOGGER.transports.Console, {
-    level:     'error',
-    colorize:  true,
-    json:      false,
-    timestamp: getTimeStamp,
-    stderrLevels: [ 'error', 'warn', 'info', 'verbose', 'debug', 'silly' ] // Which levels to send to stderr (all)
-  });
-} catch (e) {
-  console.error('Error when trying to init logging with Winston');
-  console.error(e);
-  process.exit(1);
-}
+const LOGGER = winston.createLogger({
+  level:     'error',
+  transports: [
+    new winston.transports.Console()
+  ]
+});
+
+LOGGER.configure({
+  colorize:  true,
+  json:      false,
+  stderrLevels: [ 'error', 'warn', 'info', 'verbose', 'debug', 'silly' ], // Which levels to send to stderr (all)
+  timestamp: getTimeStamp,
+});
 
 module.exports = LOGGER;
```

Comment:
The module exports the logger, but `lib/logsetup.js` is only imported in `bin/{client, server}.js`.
All the other modules make direct use of the default logger instead of the one just created, by just using `require('winston')` (instead of `require('./logsetup.js')`). I'm not sure how, but configuring this extra logger does affect the default logger. The docs don't mention this side-effect as far as I can tell, so maybe it isn't intended?

Also, the docs say the default logger doesn't have any transports, meaning that if `logsetup.js` is not run, then logging won't be output.

Reply:
I need to check which are the default for winston logger. My understanding was it will come with a default console. But likely you are right.

Reply:
Working on #245 seems to confirm that there is no default logger transport. Rebasing on this branch causes tests to fail with errors like this: `[winston] Attempt to write logs with no transports {"message":"Status code for <https://localhost:5005/>: 200","level":"debug"}`